tox_node/src/node_config.rs

```diff
@@ -36,17 +36,33 @@ impl FromStr for Threads {
 }
 
 #[cfg(unix)]
-arg_enum! {
-    /// Specifies where to write logs.
-    #[derive(Clone, Copy, PartialEq, Eq, Debug, Deserialize)]
-    pub enum LogType {
-        Stderr,
-        Stdout,
-        Syslog,
-        None,
+/// Specifies where to write logs.
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Deserialize)]
+pub enum LogType {
+    Stderr,
+    Stdout,
+    Syslog,
+    None,
+}
+
+impl ValueEnum for LogType {
```

Comment:
Maybe derive this implementation: https://docs.rs/clap/latest/clap/trait.ValueEnum.html

Reply:
We can but it’s a +1 feature.

Reply:
Yeah, but it's the purpose of this feature, and it can add dependencies only for `tox-node`. You thing we should avoid this? :)

Reply:
Why not?

Reply:
Derivation adds a bit more safety - if we need to add a new value to `LogType` without derivation we have to remember to add it to `value_variants` manually. Also there could be typos in manually written strings.

Reply:
I agree, but it's not something we touch very often.